default.yml

```diff
@@ -18,7 +18,8 @@ AllCops:
     - db/migrate/*
     - vendor/**/*
     - node_modules/**/*
-  DisabledByDefault: true
+  DisabledByDefault: false
```

Comment:
I will tidy up the configuration once we update our repos to this version of the gem since I'm expecting a few more changes.